emperor/support_files/js/controller.js

```diff
@@ -661,7 +661,24 @@ define([
                 disabled: isLargeDataset
               }
             }
-        }
+        },
+        'openInVegaEditor': {
+          name: 'Open in Vega Editor',
+          icon: 'file-picture-o',
+          callback: function(key, opts) {
+            scope.exportToVega();
+          },
+          disabled: function(key, opt) {
+            // Only enable if this is a "vanilla" plot
```

Comment:
I imagine there is a better way to identify when the plot isn't a biplot or a procrustes plot. The reason I want to disable then is because the spec is not aware of non-point glyphs (yet).